installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

```diff
@@ -17,10 +17,22 @@ include::modules/ipi-install-extracting-the-openshift-installer.adoc[leveloffset
 include::modules/ipi-install-creating-an-rhcos-images-cache.adoc[leveloffset=+1]
 
 [id="ipi-install-configuration-files"]
-== Configuration files
+== Configuring the install-config.yaml file
 
 include::modules/ipi-install-configuring-the-install-config-file.adoc[leveloffset=+2]
```

Comment:
@johnwilkins It's outside the scope of this PR, but it would be ideal if we could have a follow-up PR to adjust the heading of modules/ipi-install-configuring-the-install-config-file.adoc file to drop the back-ticks in the heading to align with the formatting here and to compliant with the [style guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings). Thanks!

Reply:
Will do.